packages/generator-one-app-module/package.json

```diff
@@ -35,5 +35,8 @@
   "devDependencies": {
     "yeoman-assert": "^3.1.1",
     "yeoman-test": "^2.1.0"
+  },
+  "publishConfig": {
+    "access": "public"
```

Comment:
I thought `publishConfig` was by default true? @nellyk @stephaniecoates what issue is this solving?

Reply:
https://github.com/lerna/lerna/tree/master/commands/publish#publishconfigaccess

`To publish packages with a scope (e.g., @mycompany/rocks), you must set access`

this code is also in all other packages besides `generator-one-app-module`.

Reply:
Yeah I see thanks.